portal-api/src/v1/users/validation/rules/surname-must-not-be-empty.test.js

```diff
@@ -0,0 +1,27 @@
+const surnameMustNotBeEmpty = require('./surname-must-not-be-empty');
+
+describe('surnameMustNotBeEmpty', () => {
+  const surnameMustNotBeEmptyError = [
+    {
+      surname: {
+        order: '2',
+        text: 'Surname is required',
+      },
+    },
+  ];
+
+  const testCases = [{ description: 'when no surname is provided', change: { surname: '' } }];
+
+  describe.each(testCases)('$description', ({ change }) => {
+    it('should return an error if no surname is provided', () => {
+      const errors = surnameMustNotBeEmpty(undefined, change);
+      expect(errors).toStrictEqual(surnameMustNotBeEmptyError);
+    });
+
+    it('should not return an error if a surname is provided', () => {
```

Comment:
This second test doesn't belong to `describe.each`

Reply:
Added `describe.each` for the above test.